src/redux/actions/portfolio-actions.js

```diff
@@ -107,3 +107,17 @@ export const selectPortfolioItem = (portfolioItem) => ({
   payload: portfolioItem
 });
 
+export const removeProductsFromPortfolio = portfolioItems => ({
+  type: ActionTypes.REMOVE_PORTFOLIO_ITEMS,
+  payload: PortfolioHelper.removePortfolioItems(portfolioItems),
+  meta: {
+    notifications: {
+      fulfilled: {
+        variant: 'success',
+        title: 'Products removed',
+        description: 'You have removed three products from the Global portfolio. Undo this if it was a mistake.'
```

Comment:
The 'three' word here needs to be removed/replaced. Either use the actual number of products removed or a general term.

Reply:
Thanks i just copied it from mocks :D

Reply:
Ok now it will print the number. We can talk about some text transformation tools later.